vision/src/autogluon/vision/predictor/predictor.py

```diff
@@ -107,7 +107,8 @@ def fit(self,
                         'model': Categorical('resnet50_v1b', 'resnet101_v1d', 'resnest200'),
                         'lr': Real(1e-5, 1e-2, log=True),
                         'batch_size': Categorical(8, 16, 32, 64, 128),
-                        'epochs': 200
+                        'epochs': 200,
+                        'early_stop_patience': -1
```

Comment:
Are we sure this is useful? Why not have it be 50? Is it realistic that it would take 100+ epochs to show improvement?

Reply:
changed best_quality to use 50 epochs, it's very reasonable for now but somewhat suboptimal comparing to `as-is` because currently the learning rate scheduler is not affected by the early stopping, in the future a LRSchedulerOnPlateau or similar stuff can offer better tradeoffs